requirements_docs.txt

```diff
@@ -38,6 +38,7 @@ mock==2.0.0
 mpmath==1.0.0
 neo-boa==0.2.1
 neo-python-rpc==0.1.8
+neo-core==0.3.6
```

Comment:
`neo-core` doesn't exist, the package is called `neocore`. Why do we need this in the docs requirements, actually?